models/endoscopic_tool_segmentation/configs/inference.json

```diff
@@ -15,7 +15,8 @@
         "backbone": "efficientnet-b2",
         "spatial_dims": 2,
         "pretrained": false,
-        "is_pad": false
+        "is_pad": false,
+        "pre_conv": null
```

Comment:
Hi @binliunls , it seems this PR has conflicts with #324 , could you please help to double confirm it? Thanks in advance!

Reply:
Hi @yiheng-wang-nv ,
        I have verified this parameter and it works well. I will also update my the #324 PR. But the `pre_conv` parameter was not added until MONAI 1.2. Maybe this one still should be a 1.2 change?

Thanks,
Bin

Reply:
Hi @binliunls , this bundle will temporarily use monai 1.2.0rc2 to support monailabel usage, thus the `pre_conv` arg can work.